src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -96,9 +103,13 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + "by: " + feedbackResponseComment.giverEmail + "<br>"
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
         }
-
-        data.comment = feedbackResponseComment;
-
+        data.comment = updatedComment;
```

Comment:
Maybe add a general comment above this that these fields are needed to correctly generate the correct `editedCommentDetails`. Also, can add a blank line to separate from the block above?